docs/advanced_topics/accessibility_considerations.rst

```diff
@@ -0,0 +1,160 @@
+Accessibility considerations
+============================
+
+Accessibility for CMS-driven websites is a matter of :ref:`modeling content appropriately <content_modeling>`, :ref:`creating accessible templates <accessibility_in_templates>`, and :ref:`authoring content <authoring_accessible_content>` with readability and accessibility guidelines in mind.
+Wagtail generally puts developers in control of content modeling and front-end markup, but there are a few areas to be aware of nonetheless, and ways to help authors be aware of readability best practices.
+
+
+* :ref:`Content modeling <content_modeling>`
+* :ref:`Accessibility in templates <accessibility_in_templates>`
+* :ref:`Authoring accessible content <authoring_accessible_content>`
+
+----
+
+.. _content_modeling:
+
+Content modeling
+~~~~~~~~~~~~~~~~
+
+As part of defining your site’s models, here are areas to pay special attention to:
+
+Alt text for images
+-------------------
+
+The default behavior for Wagtail images is to use the ``title`` field as the alt text (`#4945 <https://github.com/wagtail/wagtail/issues/4945>`_).
+This is inappropriate, as it’s not communicated in the CMS interface, and the image upload form uses the image’s filename as the title by default.
+
+Ideally, always add an optional “alt text” field wherever an image is used, alongside the image field:
+
+- For normal fields, add an alt text field to your image’s panel.
+- For StreamField, add an extra field to your image block.
+- For rich text – Wagtail already makes it possible to customize alt text for rich text images. Unfortunately it’s not currently possible to elect not to provide alt text for decorative images (see `#6494 <https://github.com/wagtail/wagtail/issues/6494>`_).
```

Comment:
"it’s not currently possible to opt out of providing alt text" removes the double negative from the sentence.

or 

"it’s not currently possible to set alt text as an optional field in these situations"